apps/dotcms-ui/src/app/api/services/dot-custom-event-handler/dot-custom-event-handler.service.ts

```diff
@@ -80,6 +82,11 @@ export class DotCustomEventHandlerService {
     }
 
     private setDotcmsUiColors($event: CustomEvent): void {
```

Comment:
We should rename this because is not just colors any more

```suggestion
    private setPersonalization($event: CustomEvent): void {
```